src/components/deliveryPaymentPage/deliveryAndPayment/DeliveryAndPayment.module.css

```diff
@@ -51,12 +51,10 @@
   background-color: #fff;
 }
 
-.paymentOption img {
-  max-width: 100%;
-  height: auto;
-}
-
 @media (max-width: 1023px) {
+  .paymentOption img {
+    max-width: 100%;
```

Comment:
I still get warnings on Delivery and Payment page for mobile version.

<img width="1285" alt="delivery-and-payment-image-warnings" src="https://github.com/user-attachments/assets/b84f2cc0-02bb-4666-927e-92d8c5587e97">

Reply:
The changes works for me! 👍  Thanks.